migrations/Version202008201138552348_generis.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\generis\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\oatbox\event\EventAggregator;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202008201138552348_generis extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'New EventAggregator Service for extension generis.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $eventAggregator = new EventAggregator(
+          [
+              'numberOfAggregatedEvents' => 10
```

Comment:
Maybe call it maxAggregatedEvent? the current name is fine as well, if you think it's better just keep it